docs/src/components/Snippets/glossary_snippets.js

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+
+export const Glossary = {
+  Tools: {
+    aztec: () => (
+      <p>
+        The CLI tool (with an extensive set of parameters) that enables users to
+        perform a wide range of tasks. It can: run a node, run a sandbox,
+        execute tests, generate contract interfaces for javascript...
```

Comment:
I think it would be good to include a link to the the CLI reference page at docs/docs/developers/reference/environment_reference/cli_reference.md here. 

It seems like using <a> tags doesn't create links to versioned docs correctly, so I think it'd be best to add the links directly in the glossary.md page instead of as snippets.

Reply:
I'm not including links in the snippets so that the choice to have a further link or not is with the destination.

Reply:
Added link beneath snippet in glossary.md

Reply:
ah makes sense :+1: